src/Farmer/Arm/Databricks.fs

```diff
@@ -0,0 +1,65 @@
+[<AutoOpen>]
+module Farmer.Arm.Databricks
+
+open Farmer
+open Farmer.Databricks
+open Farmer.Arm.Network
+open System
+
+let workspaces = ResourceType ("Microsoft.Databricks/workspaces", "2018-04-01")
+
+type ByovConfig =
+    { Vnet: ResourceRef<ByovConfig>
+      PublicSubnet: ResourceName
+      PrivateSubnet: ResourceName }
+
+type EncryptionConfig =
+    { KeySource: string
```

Comment:
This should be made into a DU. See [here](https://docs.microsoft.com/en-us/azure/templates/microsoft.databricks/workspaces#encryption-object) - there are only a couple of cases.